crates/miden-lib/src/account/faucets/mod.rs

```diff
@@ -246,7 +247,7 @@ pub fn create_basic_fungible_faucet(
 
     let auth_component: AuthRpoFalcon512Acl = match auth_scheme {
         AuthScheme::RpoFalcon512 { pub_key } => {
-            AuthRpoFalcon512Acl::new(pub_key, vec![distribute_proc_root])
+            AuthRpoFalcon512Acl::new(pub_key, vec![distribute_proc_root], false, false)
```

Comment:
Should we not set `allow_unauthorized_input_notes = true` here? I think for the faucet we wanted to enable anyone to call `burn` procedure which would require consuming an input note.